patches/server/0019-Implement-Paper-VersionChecker.patch

```diff
@@ -77,7 +79,12 @@ index 0000000000000000000000000000000000000000..1a1b50e475b9ede544b2f6d0d36632b2
 +            case -2:
 +                return Component.text("Unknown version", NamedTextColor.YELLOW);
 +            default:
-+                return Component.text("You are " + distance + " version(s) behind", NamedTextColor.YELLOW);
++                return Component.text("You are " + distance + " version(s) behind", NamedTextColor.YELLOW)
++                        .append(Component.newline())
++                        .append(Component.text("Download the new version at: ", NamedTextColor.YELLOW)
```

Comment:
I don't think you need to set the TextColor here, isn't that going to be a child component of the "You are x version(s) behind" component which already has the YELLOW color?

Reply:
Good catch, never worked with components much before.